deploy/standard/manifests/controller/helm/retina/values.yaml

```diff
@@ -98,6 +98,7 @@ volumeMounts:
   cgroup: /sys/fs/cgroup
   tmp: /tmp
   config: /retina/config
+  host-os-release: /etc/os-release
```

Comment:
This volumeMount is needed to determine if the host is Mariner, so we can decide which ebpf programs to attach. 
Retina's base image is Mariner, so the file scan of /etc/os-release would always return true otherwise.